src/test/java/net/sf/jabref/importer/fileformat/BiblioscapeImporterTest.java

```diff
@@ -0,0 +1,121 @@
+                        .getResourceAsStream("BiblioscapeImporterTest_comparefile.txt")) {
+            List<BibEntry> bScapeEntries = importer.importEntries(bScapeStream, new OutputPrinterToNull());
+            List<BibEntry> bibEntries = bibtexImporter.importEntries(bibStream, new OutputPrinterToNull());
+
+            Assert.assertEquals(bScapeEntries.size(), bibEntries.size());
+
+            int c = 0;
+            for (int i = 0; i < bScapeEntries.size(); i++) {
+                for (int j = 0; j < bibEntries.size(); j++) {
+                    BibEntry be1 = bScapeEntries.get(i);
+                    BibEntry be2 = bibEntries.get(j);
+                    if (be1.getField("isbn").equals(be2.getField("isbn"))) {
+                        Assert.assertEquals(be1.getField("abstract"), be2.getField("abstract"));
+                        Assert.assertEquals(be1.getField("address"), be2.getField("address"));
+                        Assert.assertEquals(be1.getField("annote"), be2.getField("annote"));
+                        Assert.assertEquals(be1.getField("author"), be2.getField("author"));
+                        Assert.assertEquals(be1.getField("chapter"), be2.getField("chapter"));
+                        Assert.assertEquals(be1.getField("edition"), be2.getField("edition"));
+                        Assert.assertEquals(be1.getField("journal"), be2.getField("journal"));
+                        Assert.assertEquals(be1.getField("note"), be2.getField("note"));
+                        Assert.assertEquals(be1.getField("number"), be2.getField("number"));
+                        Assert.assertEquals(be1.getField("pages"), be2.getField("pages"));
+                        Assert.assertEquals(be1.getField("pdf"), be2.getField("pdf"));
+                        Assert.assertEquals(be1.getField("publisher"), be2.getField("publisher"));
+                        Assert.assertEquals(be1.getField("title"), be2.getField("title"));
+                        Assert.assertEquals(be1.getField("url"), be2.getField("url"));
+                        Assert.assertEquals(be1.getField("volume"), be2.getField("volume"));
+                        Assert.assertEquals(be1.getField("year"), be2.getField("year"));
+                        Assert.assertEquals(be1.getField("comment"), be2.getField("comment"));
+
+                        c++;
+                    }
+                }
+            }
+
+            Assert.assertEquals(c, bScapeEntries.size());
+
+        }
+
+
+        try (InputStream is = BiblioscapeImporter.class.getResourceAsStream("BiblioscapeImporterTest1.txt")) {
+            List<BibEntry> entries = importer.importEntries(is, new OutputPrinterToNull());
+            Assert.assertEquals(17, entries.size());
+
+            for (int i = 0; i < entries.size(); i++) {
+                // Extract the bibtex entry type
+                BibEntry be = entries.get(i);
+                String type = be.getField("note").split(": ")[1];
```

Comment:
I tried to implement this but while doing I realized that it will become too complicated for a test, cause there are also entries which differ from the other.
Loop based reading (for example with `BufferedReader`) would be too dangerous, cause the sequence the importer uses to save entries is not clear.
Now I made the test a little bit safer by removing the keyword `bibtex-typ:` in field `--NT--`, so we only need to query the `note` field instead of 
splitting the string. I hope it looks a little bit better.